I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 213, in script
    python hide:
  File "renpy/common/00start.rpy", line 213, in <module>
    python hide:
  File "renpy/common/00start.rpy", line 214, in _execute_python_hide
    renpy.warp.warp()
Exception: Could not find a statement to warp to. (script.rpy:4)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 213, in script
    python hide:
  File "c:\Ren'py\renpy-8.3.4-sdk\renpy\ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "c:\Ren'py\renpy-8.3.4-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00start.rpy", line 213, in <module>
    python hide:
  File "renpy/common/00start.rpy", line 214, in _execute_python_hide
    renpy.warp.warp()
  File "c:\Ren'py\renpy-8.3.4-sdk\renpy\warp.py", line 147, in warp
    raise Exception("Could not find a statement to warp to. ({})".format(spec))
Exception: Could not find a statement to warp to. (script.rpy:4)

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.4.24120703
Freedom Project 1.0
Sun Jan 26 15:09:33 2025
